libs/ui-toolkit/src/components/toggle/toggle.stories.tsx

```diff
@@ -0,0 +1,70 @@
+import type { Story, Meta } from '@storybook/react';
+import { Toggle } from './toggle';
+import { useState } from 'react';
+
+export default {
+  component: Toggle,
+  title: 'Toggle',
+} as Meta;
+
+export const Controlled: Story = () => {
+  const [checked, setChecked] = useState('test-1');
+
+  return (
+    <Toggle
+      name="controlled"
+      toggles={[
+        {
+          label: 'Option 1',
+          value: 'test-1',
+        },
+        {
+          label: 'Option 2',
+          value: 'test-2',
+        },
+      ]}
+      checkedValue={checked}
+      onChange={(e) => setChecked(e.target.value)}
+      className="max-w-[400px]"
+    />
+  );
+};
+
+// @ts-ignore args provided after
```

Comment:
chore (blocking): I think but explicitly declaring the args type to be `ToggleInputProps` this should be removable

Reply:
Oh nice, will try that, thanks